libs/core/models/user-profile.ts

```diff
@@ -0,0 +1,21 @@
+/** User profile model. */
+export interface UserProfile {
+
+	/** Email. */
+	readonly email: string;
+
+	/** First name. */
+	readonly firstName: string;
+
+	/** Last name. */
+	readonly lastName: string;
+
+	/** Avatar. */
+	readonly avatar: string;
+
+	/** Created. */
+	readonly created: string;
```

Comment:
It doesn't convenient for us to store and handle the date in "string" format. Try now to display the date for example. We usually map the date to the Date class. Or, if you don't use this field, you can omit it in model
```suggestion
	readonly created: Date;
```

Reply:
I don't need the user for now at all, I created a model and a mapper just for consistency (maybe it is not right), so that I wouldn't return DTO from the method.

Reply:
Would it be okay if I added conversion to date to the mapper?

Reply:
Better not to add unused entities